### ISSUE TITLE
hostsblock.log lines now have strange leading text

### ISSUE THREAD

**Issue by Sadi58** [gaenserich/hostsblock#23]
After the latest commits I've begun getting the text below followed by a space before each line in the `hostsblock.log` file:
`[[0;33mINFO[0m] `
The odd character above (white question mark against black background) is displayed in my text editor as a square including `00` in the first line and `18` or `1B` in the second line (my eyes are a bit old ;-) 
I use this launcher script:
```
#!/bin/bash
/etc/hostsblock/hostsblock -v 3 &>/var/log/hostsblock.log
cat <<EOF >> /var/log/hostsblock.log
Hostsblock completed at `date +'%d/%m/%Y %H:%M'`
EOF
```

**Comment by gaenserich** [comment]
The debugging/logging beautification seems to misidentify your script as an interactive session (all the garbled text is its attempts to colorize the verbosity label. When piped to a file or when caught by systemd's logging features, it _should_ detect that it is not to use color and just read [Info].

Anyway, it should be fixed as of 2103bc47bf6f727e92dd939e06cf3b219b1fad1d

**Comment by Sadi58** [comment]
Thanks, I can confirm that I have only [INFO] and [NOTE] now.
